src/libraries/System.Security.Cryptography/tests/HmacMD5Tests.cs

```diff
@@ -15,9 +15,11 @@ public class HmacMD5Tests : Rfc2202HmacTests<HmacMD5Tests.Traits>
         public sealed class Traits : IHmacTrait
         {
             public static bool IsSupported => true;
-            public static int HashSizeInBytes => HMACSHA1.HashSizeInBytes;
+            public static int HashSizeInBytes => HMACMD5.HashSizeInBytes;
```

Comment:
This turned out to be a mistake in our HMACMD5 tests but it went unnoticed because the value is only used for "create a buffer that is at _least_ large enough". LEN(SHA1) > LEN(MD5) so it happened to work.  But let's fix it.